### PR TITLE
[Fix] Restructure Sirius in THIRDPARTY

### PR DESCRIPTION
**Pull request by axelwalter**
## Description
Moved all jar files in All/Sirius/app and put the executable files for each OS in the top level Sirius directory.

## Checklist
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes (Tick if no updates were necessary.)

### How can I get additional information on failed tests during CI
<details>
  <summary>Click to expand</summary>
If your PR is failing you can check out

- The details of the action statuses at the end of the PR or the "Checks" tab.
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. Use the "Show filters" capability on the top right to search for your PR number.
  If you click in the column that lists the failed tests you will get detailed error messages.

</details>

### Advanced commands (admins / reviewer only)
<details>
  <summary>Click to expand</summary>
  
- `/reformat` (experimental) applies the clang-format style changes as additional commit. Note: your branch must have a different name (e.g., yourrepo:feature/XYZ) than the receiving branch (e.g., OpenMS:develop). Otherwise, reformat fails to push.
- setting the label "NoJenkins" will skip tests for this PR on jenkins (saves resources e.g., on edits that do not affect tests)
- commenting with `rebuild jenkins` will retrigger Jenkins-based CI builds
  
</details>

---
:warning: Note: Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

File: src/tests/topp/THIRDPARTY/SiriusAdapter_1_output.mzTab [hunk 1]
```diff
@@ -1,60 +1,4 @@
...
-SML	null	C11H17N5O3	null	null	null	290.124002639250989	null	null	420.444000000000017	null	null	null	null	null	null	0.440145632853194	0.440145632853194	0.0	null	[M + Na]+	C11H17N5O3	3	1	0.0	5.661493518486198	5.661493518486198	5.661493518486198	49	2204006	null	sample=1 period=1 cycle=2204 experiment=6
-SML	null	C9H17N9	null	null	null	290.124002639250989	null	null	420.444000000000017	null	null	null	null	null	null	0.29900758475931	0.29900758475931	0.0	null	[M + K]+	C9H17N9	4	1	0.0	0.527344029052501	0.527344029052501	0.527344029052501	49	2204006	null	sample=1 period=1 cycle=2204 experiment=6
-SML	null	C16H17N3O	null	null	null	290.124002639250989	null	null	420.444000000000017	null	null	null	null	null	null	0.11321656639466	0.11321656639466	0.0	null	[M + Na]+	C16H17N3O	5	1	0.0	-8.204163798395776	8.204163798395776	-8.204163798395776	49	2204006	null	sample=1 period=1 cycle=2204 experiment=6
+MTD	description	null
```
Comment (jpfeuffer) [inline src/tests/topp/THIRDPARTY/SiriusAdapter_1_output.mzTab:4]: That looks very bad unfortunately

Comment (jpfeuffer) [reply src/tests/topp/THIRDPARTY/SiriusAdapter_1_output.mzTab:4]: were there changes in their folder structure/ format such that we cannot read it anymore?

Comment (axelwalter) [reply src/tests/topp/THIRDPARTY/SiriusAdapter_1_output.mzTab]: The reason for the empty files was the missing `write-summaries` flag for the Sirius 5.

**Comment by axelwalter** [comment]
rebuild jenkins

**Comment by jpfeuffer** [comment]
rebuild jenkins

**Comment by jpfeuffer** [comment]
rebuild jenkins

**Comment by jpfeuffer** [comment]
rebuild jenkins

**Comment by jpfeuffer** [comment]
rebuild jenkins

**Comment by jpfeuffer** [comment]
rebuild jenkins

**Comment by jpfeuffer** [comment]
rebuild jenkins

**Comment by jpfeuffer** [comment]
rebuild jenkins

**Comment by axelwalter** [comment]
rebuild jenkins

**Comment by jpfeuffer** [comment]
rebuild jenkins

**Comment by jpfeuffer** [comment]
rebuild jenkins

**Review by jpfeuffer — APPROVED** [review]

**Review by jpfeuffer — APPROVED** [review]

**Comment by jpfeuffer** [comment]
TODO
- double-check login
- get unzipped summaries?
- double-check if the new test results make sense. They are sometimes quite different.

Sirius and AGM tests are globally deactivated for now. But this is a necessary step in the right direction, so
I will merge.

**Pull request merged**